api/features/views.py

```diff
@@ -73,6 +74,22 @@
 flags_cache = caches[settings.FLAGS_CACHE_LOCATION]
 
 
+@swagger_auto_schema(responses={200: ListCreateFeatureSerializer()}, method="get")
+@api_view(["GET"])
+@permission_classes([IsAuthenticated | HasMasterAPIKEY])
+def get_feature_by_uuid(request, uuid):
+    if request.user.is_anonymous:
```

Comment:
We should use the existence of the master api key here rather than checking if the user is anonymous.